### PR TITLE
Refine the doc and unit test for Sigmoid and stanh

### PR DESCRIPTION
**Pull request by hong19860320**
### PR types
Others

### PR changes
APIs

### Describe
1. Remove 'disable_static' and 'print' statements in Sigmoid examples
2. Fix the example and doc of stanh 

image
image

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -9520,36 +9520,36 @@ def pow(x, factor=1.0, name=None):
...
-            output = paddle.stanh(data, scale_a=0.67, scale_b=1.72)
-            print(data)
-            # [[0.19412413, 0.66871136, 0.77059180],
-            #  [0.89738929, 0.35827777, 0.60592669],
-            #  [0.66346580, 0.78424633, 0.46533889]]
-            print(output)
-            # [[0.22245567, 0.72288811, 0.81671900],
-            #  [0.92525512, 0.40512756, 0.66227961],
-            #  [0.71790355, 0.82885355, 0.51953089]]
+            x = paddle.to_tensor(np.array([1.0, 2.0, 3.0, 4.0], 'float32'))
```
Comment (zhupengyang) [inline python/paddle/fluid/layers/nn.py]: 可以传list，不需要numpy

File: python/paddle/nn/layer/activation.py [hunk 2]
```diff
@@ -554,12 +554,9 @@ class Sigmoid(layers.Layer):
           import numpy as np
           import paddle
 
-          paddle.disable_static()
-          input_data = np.array([1.0, 2.0, 3.0, 4.0]).astype('float32')
           m = paddle.nn.Sigmoid()
-          x = paddle.to_tensor(input_data)
-          output = m(x)
-          print(output.numpy()) # [0.7310586, 0.880797, 0.95257413, 0.98201376]
+          x = paddle.to_tensor(np.array([1.0, 2.0, 3.0, 4.0], 'float32'))
```
Comment (zhupengyang) [inline python/paddle/nn/layer/activation.py]: 同上，可以不需要numpy

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (hong19860320) [reply python/paddle/fluid/layers/nn.py]: DONE

File: python/paddle/nn/layer/activation.py (hunk 2 above)
Comment (hong19860320) [reply python/paddle/nn/layer/activation.py]: DONE

**Review by zhupengyang — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**